css/src/components/menu.module.css

```diff
@@ -13,6 +13,13 @@
   box-sizing: border-box;
 }
 
+.Menu-Item--small {
+  padding: var(--spacing-10) var(--spacing-20);
+  font-size: var(--font-size-s);
+  font-weight: var(--font-weight-medium);
+  line-height: var(--font-height-s);
```

Comment:
Menu Item should not control the font-size etc. of the content which is rendered inside it. User can directly use `<Text>` component inside MenuItem based on their requirements.